server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java

```diff
@@ -129,11 +129,12 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     }
 
     private void addContentDispositionHeader(Optional<String> optionalName, HttpServletResponse resp) {
-        optionalName.ifPresent(name -> resp.addHeader("Content-Disposition", "attachment; filename=\"" + name + "\""));
```

Comment:
The filename disappear from the content-disposition header only because we can't handle non ASCII characters in this header. But it's still appearing in the Download endpoint.